### PR TITLE
Docs: Fix broken link

### PR DESCRIPTION
**Pull request by Clivern**
**What this PR does / why we need it**: Fixes a broken link in the documentation

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**: I'm guessing the right link in this PR

### CODE REVIEW

**Review by Clarity-89 — APPROVED** [review]

File: docs/sources/setup-grafana/configure-security/configure-authentication/auth-proxy.md [hunk 1]
```diff
@@ -308,4 +308,4 @@ a login token and cookie. You only have to configure your auth proxy to provide
 Requests via other routes will be authenticated using the cookie.
 
 Use settings `login_maximum_inactive_lifetime_duration` and `login_maximum_lifetime_duration` under `[auth]` to control session
-lifetime. [Read more about login tokens]({{< relref "/#login-and-short-lived-tokens" >}})
+lifetime. [Read more about login tokens]({{< relref "../#login-and-short-lived-tokens" >}})
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/auth-proxy.md:311]:
```suggestion
lifetime. [Read more about login tokens]({{< relref "./#login-and-short-lived-tokens" >}}).
```

Comment (chri2547) [reply docs/sources/setup-grafana/configure-security/configure-authentication/auth-proxy.md:311]: Small adjustment to the path. I tested and what I have works now. Thank you very much for catching this!

**Review by chri2547 — COMMENTED** [review]
small adjust. Please accept suggestion and then I'll approve.

**Review by chri2547 — APPROVED** [review]
Awesome, thank you for your contribution!

**Review by gguillotte-grafana — APPROVED** [review]

**Pull request merged**